bin/live/pycbc_live_combine_single_fits

```diff
@@ -102,16 +105,16 @@ for f in args.trfits_files:
                 logging.info(fits_f[ifo + '/fit_coeff'][:])
     fits_f.close()
 
-# Set up the date array, this is stored as an offset from the start date of
-# the combination to the end of the trigger_fits file.
-# This allows for missing or empty days
+# Set up the date array, this is stored as an offset from the first trigger time of
+# the first file to the last trigger of the file
 
-trigger_file_times = np.array(trigger_file_times)
-ad_order = np.argsort(np.array(trigger_file_times))
-start_date_n = trigger_file_times[ad_order[0]]
-ad = trigger_file_times[ad_order] - start_date_n
+trigger_file_starts = np.array(trigger_file_starts)
+trigger_file_ends = np.array(trigger_file_ends)
+ad_order = np.argsort(np.array(trigger_file_starts))
+start_date_n = trigger_file_starts[ad_order[0]]
+ad = trigger_file_ends[ad_order] - start_date_n
 
-# Get the counts and alphas sorted by bin rather than by date
+# Get the counts and alphas sorted by date
```

Comment:
If I understand lines 157-158 right, these are *not* sorted by date, right? Is that not the wrong assumption at the origin of the bug?

Reply:
Ah, I guess the comment is meant to also describe what happens in lines 157-158?